modules/documents/client/views/partials/modal-document-confirm-publish.html

```diff
@@ -3,10 +3,12 @@
 		<button class="btn btn-default close" type="button" ng-click="confirmDlg.cancel()">
 			<span aria-hidden="true">&times;</span>
 		</button>
-		<h3 class="modal-title">Confirm Publish File(s)</h3>
+		<h3 ng-if="confirmDlg.publishableFiles.length === 0">No files can be published</h3>
```

Comment:
Too many ng-ifs.  You're better off using filters to achieve the desired result.